tf2onnx/rewriter/dropout_rewriter.py

```diff
@@ -37,21 +40,29 @@ def rewrite_dropout(g, ops):
             OpTypePattern("Cast", inputs=[
                 OpTypePattern("GreaterEqual", inputs=[
                     OpTypePattern("RandomUniform|RandomUniformLike"),
-                    OpTypePattern(None, name="input3")
+                    OpTypePattern("*", name="input3")
                 ])
             ]),
             OpTypePattern("Mul", name="input2"),
         ]),
     ]
     for pattern in patterns:
-        matcher = GraphMatcher(pattern, allow_reorder=True)
+        matcher = GraphMatcher(pattern, allow_reorder=False)
```

Comment:
There is a Add in the pattern ... tensorflow could switch the inputs

Reply:
There is also a GreaterEqual for which switching the inputs would be incorrect.  Finding such a pattern is unlikely but possible, and would not be equivalent to a Dropout node.  I'll switch allow_reorder to True for now, and fix it with #998

Reply:
there is a pattern as input to GreaterEqual which prevents it in this case from being switched. Basically tensorflow can't switch the Random* with the 2nd input and we are looking for it so we can't switch the inputs either.
Not saying that the reorder can't be an issue in other cases.